### PR TITLE
Add DOIs for tesseroid variable density paper

### LINKED ISSUE #69: Update variable density tesseroid paper
Update the paper entry with the DOIs for the supplement, official version, and preprint.

### PR DESCRIPTION
**Pull request by santisoler**
Fix #69

### CODE REVIEW

File: papers/tesseroid-variable-density.md [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
+date: 2019-06-05
 author: santisoler, agustina, uieda, mario
 repository: pinga-lab/tesseroid-variable-density
 journal: Geophysical Journal International
+supplement: 10.6084/m9.figshare.8239622
+preprint: 10.31223/osf.io/3548g
+doi: 10.1093/gji/ggz277
+citation: "Soler, S. R., Pesce, A., Gimenez, M. E., & Uieda, L., 2019. Gravitational field calculation in spherical coordinates using variable densities in depth, Geophysical Journal International, doi:10.1093/gji/ggz277"
 related_papers: paper-tesseroids-2016
 alm: false
```
Comment (santisoler) [inline papers/tesseroid-variable-density.md]: @leouieda I don't know that `alm` stands for and what does it do. Feel free to set it to `true` if needed.

File: papers/tesseroid-variable-density.md [hunk 2]
```diff
@@ -1,20 +1,27 @@
 ---
 title: "Gravitational field calculation in spherical coordinates using variable densities in depth "
-date: 2019-07-01
+date: 2019-06-05
```
Comment (santisoler) [inline papers/tesseroid-variable-density.md]: This is the acceptance date. Should I change it to the publication date (11 June 2019)?

File: papers/tesseroid-variable-density.md (hunk 1 above)
Comment (santisoler) [reply papers/tesseroid-variable-density.md]: I suspect that `alm` == "Article Level Metrics"... am I right?

Comment (leouieda) [reply papers/tesseroid-variable-density.md]: Yes!

File: papers/tesseroid-variable-density.md (hunk 2 above)
Comment (leouieda) [reply papers/tesseroid-variable-density.md]: Yes we should

**Comment by santisoler** [comment]
I should update the abstract figure because it's the old one.

File: papers/tesseroid-variable-density.md (hunk 1 above)
Comment (santisoler) [reply papers/tesseroid-variable-density.md]: Done! :+1:

File: papers/tesseroid-variable-density.md (hunk 2 above)
Comment (santisoler) [reply papers/tesseroid-variable-density.md]: Done! :+1:

**Comment by santisoler** [comment]
@leouieda Now I think it's ready to be merged!

**Comment by santisoler** [comment]
@leouieda Now I think it's ready to be merged!

**Comment by leouieda** [comment]
:+1: sorry these notifications fell through the cracks. Merging!

**Pull request merged**